content/en/docs/release-notes/release-notes-0.14.md

```diff
@@ -0,0 +1,122 @@
+| OpenShift 4          | `cert-manager.yaml`        |
+| Kubernetes 1.11-1.14 | `cert-manager-legacy.yaml` |
+| OpenShift 3.11       | `cert-manager-legacy.yaml` |
+
+Please be sure to read the upgrade guide for more information on how to upgrade from a previous release.
+
+## `CustomResourceDefinition` conversion webhook + `v1alpha3` API version
+
+As part of the effort to mature our API, we are releasing the `v1alpha3` API version. This contains a number of small
+changes, notably moving some fields to the `subject` stanza on the Certificate resource to be more consistent with how
+certain options are specified.
+
+With this we have enabled the 'conversion webhook', which enables API clients to utilize both the `v1alpha2` and
+`v1alpha3` APIs simultaneously, similar to other core resources in Kubernetes.
+
+Thanks to this conversion webhook, this upgrade and future upgrades after it should be seamless. The ability to make
+these kinds of changes to our API will enable the `v1beta1` API version to be released in a seamless manner in an
+upcoming release too.
+
+## Support for Kubernetes 1.11 and OpenShift 3.11
+
+We've had a number of users who are using OpenShift 3.11 & Kubernetes 1.11 reach out requesting support with
+installation. In this release, we've expanded the range of Kubernetes versions we support to once again include 1.11,
+as well as adding support for OpenShift 3.11.
+
+A big thanks to [`@meyskens`](https://github.com/meyskens) for putting this together!
+
+## Experimental 'bundle format' support (JKS and PKCS#12)
+
+One of our top feature requests has been for support for [JKS and PKCS#12](https://github.com/jetstack/cert-manager/issues/586)
+bundle files as an output from Certificate resources.
+
+In this release, we've added experimental support for both of these bundle formats. This can currently only be
+configured globally with flags provided to the `cert-manager` pod (`--experimental-issue-jks` and
+`--experimental-issue-pkcs12`). The password used for this bundle must _also_ be configured using the flags
+`--experimental-jks-password` and `--experimental-pkcs12-keystore-password` respectively.
+
+In the next release, we are aiming to provide native support for these bundle format types as part of the Certificate
+resource configuration. We have added these flags now in order to gather feedback on the way this feature works, and
+help guide how this feature should work in future.
+
+## Extended support for Venafi features
+
+Users of the Venafi issuer often need to set custom metadata on their certificate requests in order to better associate
+each request with different business areas, or in order to validate & authorize whether a request should be signed.
+
+In this release, we've added support for setting custom metadata by adding the `venafi.cert-manager.io/custom-fields`
+annotation on `Certificate` and `CertificateRequest` resources. If using the Venafi TPP integration, version 19.2 or
+greater is required.
```

Comment:
```suggestion
In this release, we've added support for setting custom metadata by adding the `venafi.cert-manager.io/custom-fields`
annotation on `Certificate` and `CertificateRequest` resources. 
This feature requires Venafi TPP >= v19.2.
```

Reply:
I'd worded it this way because if they are using Venafi Cloud, this feature 'just works' - I didn't want to imply that the feature *requires* TPP, as this is not the case (but if TPP _is_ used, 19.2 is the minimum version)